kernel/os/src/os.c

```diff
@@ -235,8 +232,8 @@ os_init(int (*main_fn)(int argc, char **arg))
     if (main_fn) {
         err = os_task_init(&g_os_main_task, "main", os_main, main_fn,
                    OS_MAIN_TASK_PRIO,
-                   (OS_MAIN_TASK_TIMER_TICKS == 0) ? \
-                        OS_WAIT_FOREVER : OS_MAIN_TASK_TIMER_TICKS,
+                   (OS_MAIN_TASK_SANITY_ITVL_TICKS == 0) ? \
+                        OS_WAIT_FOREVER : OS_MAIN_TASK_SANITY_ITVL_TICKS,
```

Comment:
No need for \ here btw. Should work without it. Looks fine otherwise.

Reply:
I spaced it to two lines to keep it under 80 characters per line tho I'd have to admit, I haven't read the style guide for this project. I'll remove the `\`.